packages/core/lib/core/realm.js

```diff
@@ -1,17 +1,23 @@
 import Realm from 'realm';
-import {Account, TokenPrice, Transaction} from './realm-schemas';
+import {Account, TokenPrice} from './realm-schemas';
 
 let realm;
 
+const schema = [TokenPrice, Account];
+
+export function addSchema(item) {
+  schema.push(item);
+}
+
 export async function initRealm() {
   realm = await Realm.open({
     path: 'dock',
-    schema: [TokenPrice, Transaction, Account],
+    schema,
     schemaVersion: 3,
-    deleteRealmIfMigrationNeeded: false,
-    migration: () => {
-      // No migration required so far
-    },
+    deleteRealmIfMigrationNeeded: true,
```

Comment:
this is safe to do because all the data in Realm is just a local cache of data available on the chain, right?

Reply:
Yes, but I will push an additional PR adding data migrations. Then we can turn off this flag.